mm2src/coins/eth.rs

```diff
@@ -821,96 +827,94 @@ impl SwapOps for EthCoin {
         amount: &BigDecimal,
         min_block_number: u64,
         _uuid: &[u8],
-    ) -> Box<dyn Future<Item = (), Error = String> + Send> {
+    ) -> Box<dyn Future<Item = (), Error = MmError<String>> + Send> {
```

Comment:
There is no reason to replace `String` error with `MmError<String>`. It's not informative.
Consider adding a corresponding error. I'm not sure, but it could be `ValidateFeeError`, for example. Or it could be `ValidateSwapTxError` that can be used in other `SwapOps` validation methods.

Reply:
ok cool. thanks